website/pages/en/versions.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+
+const CompLibrary = require('../../core/CompLibrary');
+const Container = CompLibrary.Container;
+
+const CWD = process.cwd();
+
+const siteConfig = require(CWD + '/siteConfig.js');
+const versions = require(CWD + '/versions.json');
+
+class Versions extends React.Component {
+  render() {
+    const latestVersion = versions[0];
+    return (
+      <div className="docMainWrapper wrapper">
+        <Container className="mainContainer versionsContainer">
+          <div className="post">
+            <header className="postHeader">
+              <h2>{siteConfig.title + ' Versions'}</h2>
+            </header>
+            <a name="latest" />
```

Comment:
Anchors should have a `href` attribute, you can use a button instead. Please fix this in other places too,

Reply:
That page was auto-generated by Docusaurus and I think it's intended. I think it should act as an in-page anchor so one can link to that part of the page directly, like this: https://facebook.github.io/relay/en/versions.html#latest.

Reply:
Ah, this is so wrong. The `name` attr on anchors is obsolete. They should just use `id`s. Mind filing an issue or PR fixing that? There's no need to create artificial pseudo anchor too.

Reply:
I opened a PR for that as it's pretty straightforward ([here](https://github.com/facebook/Docusaurus/pull/449)). I also updated this PR by removing the redundant anchor tags and using `id`s instead.